apps/src/aichat/views/WaitingAnimation.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import moduleStyles from './WaitingAnimation.module.scss';
+
+export const WaitingAnimation: React.FunctionComponent<{
```

Comment:
I wonder if this is something we could put in the aiComponentLibrary? 

@etaderhold @cearachew What do you think about adding to the TA chat experience?